### PR TITLE
Fix pthid random value in get_did_exchange_message

### PR DESCRIPTION
**Pull request by vineetpant**
## Description 

Pthid value in `get_did_exchange_message` function is currently filled with random UUID value, this should remain None if not supplied

## Details

- Fix pthid value to be optional

### CODE REVIEW

**Review by Tschuck — APPROVED** [review]

File: src/protocols/did_exchange/helper.rs [hunk 1]
```diff
@@ -147,9 +147,7 @@ pub fn get_did_exchange_message(
                 .or(Some(&fallback_id))
                 .map(|v| v.to_owned()),
             other: HashMap::new(),
-            pthid: message
-                .pthid
-                .or_else(|| Some(format!("{fallback_id}#key-1"))),
+            pthid: message.pthid,
```
Comment (wulfraem) [inline src/protocols/did_exchange/helper.rs:150]: Can `fallback_id` creation now be removed? ^^

**Review by wulfraem — CHANGES_REQUESTED** [review]

**Comment by wulfraem** [comment]
And we have a merge conflict. :D

**Comment by vineetpant** [comment]
> And we have a merge conflict. :D

Done , resolved the conflicts

File: src/protocols/did_exchange/helper.rs (hunk 1 above)
Comment (wulfraem) [reply src/protocols/did_exchange/helper.rs:155]: Ah, there it is, search didn't light up on that line. Sorry for the confusion. >.<

**Pull request merged**